src/services/usersService.ts

```diff
@@ -0,0 +1,109 @@
+import { Disposable } from "vscode";
+import { AltimateRequest } from "../altimate";
+import { DBTTerminal } from "../dbt_client/dbtTerminal";
+import { provideSingleton } from "../utils";
+import { DBTProjectContainer } from "../manifest/dbtProjectContainer";
+import { DBTInstallationVerificationEvent } from "../dbt_client/dbtVersionEvent";
+
+export interface TenantUser {
+  id: number;
+  uuid: string;
+  display_name: string;
+  first_name: string;
+  last_name: string;
+  email: string;
+  phone: string;
+  is_active: boolean;
+  is_verified: boolean;
+  is_invited: boolean;
+  is_onboarded: boolean;
+  created_at: string;
+  role_title: string;
+}
```

Comment:
This seems more of an altimate thing. Something coming back from backend service. Should it be moved to altimate.ts

Reply:
fixed